utils/sysfs/sysfs.go

```diff
@@ -37,9 +38,16 @@ const (
 	ppcDevTree   = "/proc/device-tree"
 	s390xDevTree = "/etc" // s390/s390x changes
 
-	coreIDFilePath    = "/topology/core_id"
-	packageIDFilePath = "/topology/physical_package_id"
-	meminfoFile       = "meminfo"
+	coreIDFilePath       = "/topology/core_id"
```

Comment:
nit: suggest to reorder the consts and list them something like this to make it a bit more clear

```
sysFsCPUTopology     = "topology"

CPUCoreID            = "core_id"
CPUPhysicalPackageID = "physical_package_id"

coreIDFilePath       = "/" + sysFsCPUTopology "/" + CPUCoreID
packageIDFilePath    = "/" + sysFsCPUTopology + "/" + CPUPhysicalPackageID
```